server/v2/cometbft/abci.go

```diff
@@ -67,6 +68,11 @@ type Consensus[T transaction.Tx] struct {
 	extendVote             handlers.ExtendVoteHandler
 	checkTxHandler         handlers.CheckTxHandler[T]
 
+	// optimisticExec contains the context required for Optimistic Execution,
+	// including the goroutine handling.This is experimental and must be enabled
```

Comment:
let's always enable it, and remove the way to disable it (cc @tac0turtle)

Reply:
sounds good to me